### PR TITLE
Add config for github.com/kubernetes target-org

### PR DESCRIPTION
**Pull request by sttts**
This is the "production" config for the bot pushing directly into github.com/kubernetes.

### CODE REVIEW

**Comment by sttts** [comment]
@caesarxuchao do you have anything that we have to do or add before enabling this? I would do so early next week if you agree.

File: configs/kubernetes-nightly-configmap.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ metadata:
 data:
   config: |
     target-org: kubernetes-nightly
-    github-issue: 56876
-    dry-run: false
\ No newline at end of file
+    # github-issue: 56876
```
Comment (caesarxuchao) [inline configs/kubernetes-nightly-configmap.yaml]: This was to test if the issue reporter was working, right? Can we delete it now?

File: configs/kubernetes-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+kind: ConfigMap
+metadata:
+  name: publisher-config
+data:
+  config: |
+    target-org: kubernetes
+    github-issue: 56876
```
Comment (caesarxuchao) [inline configs/kubernetes-configmap.yaml]: Remove it?

**Comment by caesarxuchao** [comment]
> @caesarxuchao do you have anything that we have to do or add before enabling this?

No, I'm excited to see the bot get resurrected!

Though I'll be OOO until the new year, so I won't be able to help solving issues if there is any.

File: configs/kubernetes-nightly-configmap.yaml (hunk 1 above)
Comment (sttts) [reply configs/kubernetes-nightly-configmap.yaml]: Yes, the nightly config will go away.

File: configs/kubernetes-configmap.yaml (hunk 2 above)
Comment (sttts) [reply configs/kubernetes-configmap.yaml]: This is the issue to post logs to

**Comment by sttts** [comment]
> Though I'll be OOO until the new year, so I won't be able to help solving issues if there is any.

I am available most of the time for urgent issues. Moreover, we have @munnerz and @marun to get involved :)

**Comment by caesarxuchao** [comment]
lgtm

**Pull request merged**